libs/data-mapper/src/lib/components/inputDropdown/InputDropdown.tsx

```diff
@@ -39,7 +39,7 @@ const customValueDebounceDelay = 300;
 interface SharedOptionData {
```

Comment:
I think you may need to add the .json file extension to the file browse button wherever it is in the ConfigPanel stuff as well if you haven't already

Reply:
https://github.com/Azure/LogicAppsUX/pull/1755/files#diff-066ad794c957583c3272618fe1455d04ee2c3927884bbe93eb17ce238f7520afR103 This?